### PR TITLE
add jest and cypress badges

### PR DESCRIPTION
**Pull request by InnocenceD**
## Checklist

- [N/A] Did you write tests for your code?
- [x] Did all the tests pass locally?

## Trello Ticket

N/A

## Short Description

Add Jest and Cypress badge in readme

## Reviewer Notes

Anything special you want the reviewer to know?

### CODE REVIEW

**Review by Joyfal — APPROVED** [review]

**Review by Joyfal — CHANGES_REQUESTED** [review]
One question: should we also add badges with Cypress?

**Comment by InnocenceD** [comment]
> One question: should we also add badge with Cypress?

It will be in a different PR if we want.

**Comment by Joyfal** [comment]
> > One question: should we also add badge with Cypress?
> 
> It will be in a different PR if we want.

I think we can add that in this PR, no need to have separate PRs for small things like this.

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "build:dev": "env-cmd -f .env.dev yarn run build",
     "build:main": "env-cmd -f .env.main yarn run build",
     "cy:run": "yarn run cleannyc && yarn run cy:run:dev",
-    "cy:run:dev": "yarn run cleannyc && env-cmd -f .env.dev cypress run --browser chrome --headless",
+    "cy:run:dev": "yarn run cleannyc && env-cmd -f .env.dev cypress run --browser chrome --headless --record --key 47614646-0a12-4408-ae45-4117b3a255f0",
```
Comment (Joyfal) [inline package.json:57]: I don't think we want to update the status here. Should add it in our workflow.

**Review by Joyfal — CHANGES_REQUESTED** [review]

**Review by Joyfal — APPROVED** [review]
Awesome stuff!

**Pull request merged**